src/dia/Paper.mjs

```diff
@@ -2272,15 +2272,13 @@ export const Paper = View.extend({
                 this._mw_evt_buffer.deltas.push(deltaY);
                 this._processMouseWheelEvtBuf();
             }
-        } else if(Math.abs(deltaX) != 0 && Math.abs(deltaY) != 0) {
-            // This is a 2-axis-scroll gesture, we must not debounce the event
-            this.trigger('paper:pan', evt, deltaX, deltaY);
         } else {
             const delta = Math.max(-1, Math.min(1, originalEvent.wheelDelta));
             if (view) {
                 view.mousewheel(evt, localPoint.x, localPoint.y, delta);
 
             } else {
+                this.trigger('paper:pan', evt, deltaX, deltaY);
```

Comment:
`paper:pan` should be triggered on views too (keeping the same signature without the cellView).

Reply:
Added docs:

![SCR-20220912-fa5](https://user-images.githubusercontent.com/89727/189614900-182cc81e-0977-4a51-b74b-ab0add0e0cdf.png)